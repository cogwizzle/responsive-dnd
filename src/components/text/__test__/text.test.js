import React from 'react';
import Renderer from 'react-test-renderer';
import Text from '../text.component';

it('Render', () => {
  const dom = Renderer.create(
    <Text />
  );

  expect(dom.toJSON()).toMatchSnapshot();
});
