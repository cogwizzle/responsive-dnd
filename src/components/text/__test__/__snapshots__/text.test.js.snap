// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render 1`] = `
<div
  className="text"
>
  <input
    className="text__input "
    type="text"
  />
</div>
`;
